html/changelogs/omicega-fdsfidsofidosfgmidogfdg.yml

```diff
@@ -0,0 +1,12 @@
+author: Omicega
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - tweak: "Removed human slowdown at higher hunger and thirst levels."
```

Comment:
May wish to just remove "human" here as it may mislead people into thinking it's only for humans and not other species.

Reply:
I did this for clarity's sake. Thanks for the suggestion!